From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ryan Volz <ryan.volz@gmail.com>
Date: Mon, 12 Feb 2024 12:45:00 -0500
Subject: [PATCH] Don't build a resource file for the executable on Windows.

conda-forge is not set up with an RC compiler.
---
 src/CMakeLists.txt | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 845b196..f0dcdb1 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -53,19 +53,6 @@ else()
     )
 endif()
 
-###############################################################################
-# Resource file - adds an icon to GQRX executable
-if (WIN32)
-    set(ICON_SOURCE ${CMAKE_SOURCE_DIR}/resources/icons/gqrx.ico)
-    set(RES_FILES "${CMAKE_CURRENT_BINARY_DIR}/gqrx.rc")
-    file(WRITE "${RES_FILES}" "id ICON \"${ICON_SOURCE}\"")
-    set(CMAKE_RC_COMPILER_INIT windres)
-    enable_language(RC)
-    set(CMAKE_RC_COMPILE_OBJECT
-        "<CMAKE_RC_COMPILER> <FLAGS> -O coff <DEFINES> -i <SOURCE> -o <OBJECT>")
-    list(APPEND RESOURCES_LIST ${RES_FILES})
-endif(WIN32)
-
 ###############################################################################
 # Build the program
 add_executable(${PROJECT_NAME} ${${PROJECT_NAME}_SOURCE} ${UIS_HDRS} ${RESOURCES_LIST})
-- 
2.42.0

